net-mgmt/zabbix-proxy4/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		zabbix-proxy4
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		Zabbix Proxy enables decentralized monitoring
+PLUGIN_DEPENDS=		zabbix4-proxy
```

Comment:
@fichtner is there a PLUGIN_CONFLICTS? If yes, it should conflict with the old one.

Reply:
Like with frr -> quagga, good idea! :)